File: foundation/foundation 2 month/lesson8/homework/ex_4.c
#include <stdio.h>
int check(int k)
{
	if(k==1 || k==0)
	return  k;
	return k*check(k-1);

}

int main()
{
	int n;
	printf("N ni kiriting: ");
	scanf("%d",&n);
	int d=check(n);
	printf("%d", d);
}

File: foundation/foundation 3 month/lesson5/homework/ex_4.py
import random
ms=set()
k=int(input("K ni kiriting: "))
for x in range(k):
	ms.add(random.randint(10,99))
print("Setning elmentlari: ",ms)
arr=list(ms)
j=sorted(arr)
d=j.pop(0)
check=set(j)
print("Uchilgandan keyin: ",check)
